build.gradle

```diff
@@ -47,16 +47,18 @@ ext {
     minSdkVersion = 15
     targetSdkVersion = 28
     supportLibVersion = '1.1.0'
+    recyclerViewVersion = '1.2.0-alpha02'
     constraintLayoutVersion = '1.1.3'
-    materialVersion = '1.0.0'
+    materialVersion = '1.1.0'
     archComponentsVersion = '2.2.0'
-    roomVersion = '2.2.3'
-    pagingVersion = '2.1.0-alpha01'
-    retrofitVersion = "2.6.0"
-    okhttpLoggingInterceptorVersion = "4.0.0"
+    roomVersion = '2.2.4'
+    pagingVersion = '3.0.0-alpha01'
+    retrofitVersion = '2.6.0'
+    okhttpLoggingInterceptorVersion = '4.0.0'
+    coroutines = '1.3.0'
```

Comment:
I think this is quite old, current is 1.3.70

Reply:
the kotlin version is 1.3.70, the coroutines version [seems to be 1.3.4](https://github.com/Kotlin/kotlinx.coroutines)